subprojects/docs/src/snippets/userguide/dependencyManagement/customizingResolution/metadataRule/failRuntimeClasspathResolve.out

```diff
@@ -4,50 +4,28 @@ Execution failed for task ':failRuntimeClasspathResolve'.
      Required by:
          project :
       > Cannot choose between the following variants of org.lwjgl:lwjgl:3.2.3:
-          - enforced-platform-runtime
           - natives-windows-runtime
           - natives-windows-x86-runtime
-          - platform-runtime
```

Comment:
I was wondering about this when I added this test, because I think it was confusing that the "platform" variants are in this error. Now I know why, and it is much better now.